apps/fft/CMakeLists.txt

```
@@ -28,13 +28,23 @@ add_halide_library(fft_inverse_c2c FROM fft.generator
                    GENERATOR fft
                    PARAMS direction=frequency_to_samples size0=16 size1=16 input_number_type=complex output_number_type=complex)
 
+# Use a small stack, to stress out the compiler and act as a canary
+# for stack bloat problems. This can be accomplished with a linker
+# flag on os x, and testing this on one platform is sufficient.
+set(LDFLAGS "")
+if (CMAKE_SYSTEM_NAME MATCHES "Darwin")
+    set(LDFLAGS "-Wl,-stack_size;-Wl,0x100000")
```

For future-proofing portability.
```suggestion
    set(LDFLAGS "LINKER:-stack_size,0x100000")
```

---

Done